scripts/update_api.py

```diff
@@ -93,7 +93,7 @@ def is_fn(ty):
 # Mapping to ML types
 Type2ML = { VOID : 'unit', VOID_PTR : 'ptr', INT : 'int', UINT : 'int', INT64 : 'int', UINT64 : 'int', DOUBLE : 'float',
             FLOAT : 'float', STRING : 'string', STRING_PTR : 'char**',
-            BOOL : 'bool', SYMBOL : 'z3_symbol', PRINT_MODE : 'int', ERROR_CODE : 'int', CHAR : 'char', CHAR_PTR : 'string', LBOOL : 'uint' }
+            BOOL : 'bool', SYMBOL : 'z3_symbol', PRINT_MODE : 'int', ERROR_CODE : 'int', CHAR : 'char', CHAR_PTR : 'string', LBOOL : 'int' }
```

Comment:
Z3_lbool is defined as an enum in z3_api.h. All enum types are translated uniformly. Is there an issue with this approach?

Reply:
Probably not. I thought it would  be more consistent with the existing code, as PRINT_MODE and ERROR_CODE also have their own definitions.

Reply:
Correction: It seems to be necessary. I cannot use UINT/INT as the Python script does not add a cast otherwise. It works for the other enums that do not have definitions in the Python script, as they are only used as return types and not as arguments